From df5ccd3adbd510b033a13221f35f5434b3325095 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 27 Feb 2020 19:12:21 +0000
Subject: [PATCH 365/462] Mantis 0056312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

add environment variables DIR_XSD, INC_XSD, XSD_COMP


git-svn-id: svn://localhost/SelfServiceBuild/trunk@671 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 26d9285..f5625c1 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -405,9 +405,9 @@ set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
 set DIR_SED=%DIR_GLOBALSLIB%\sed\0.9.9.9\win32
 set SED=%DIR_SED%\bin\sed.exe
 
-set XSD_DIR=%DIR_GLOBALSLIB%\xsd\4.0.0\win
-set INC_XSD=-I%XSD_DIR%\libxsd
-set XSD_COMP=%XSD_DIR%\bin\xsd.exe
+set DIR_XSD=%DIR_GLOBALSLIB%\xsd\4.0.0\win
+set INC_XSD=-I%DIR_XSD%\libxsd
+set XSD_COMP=%DIR_XSD%\bin\xsd.exe
 
 
 if "%USE_SPRIT%"=="1.6.1" (
-- 
2.41.0.windows.1

